src/scss/style.scss

```diff
@@ -541,7 +541,7 @@ a:hover, a:active {
 
 .main-navigation ul li:hover > ul,
 .main-navigation ul li.focus > ul {
-  left: auto;
+  left: -70%;
```

Comment:
There seem to be some issues positioning the menu:
<img width="335" alt="Sub-menu not aligned with the item" src="https://user-images.githubusercontent.com/6073968/37090367-00c194f6-21fc-11e8-9cb0-16d0a714cc6b.png">
and the border should be the length of the text, I guess:
<img width="278" alt="Border longer than the text" src="https://user-images.githubusercontent.com/6073968/37090394-1b249b2c-21fc-11e8-8877-d923eebfdaee.png">

Reply:
Thanks! i'll fix it